Mathlib/FieldTheory/IsAlgClosed/Basic.lean

```diff
@@ -409,44 +409,38 @@ theorem Algebra.IsAlgebraic.range_eval_eq_rootSet_minpoly (x : K) :
   range_eval_eq_rootSet_minpoly_of_splits A (fun _ ↦ IsAlgClosed.splits_codomain _) hK x
 #align algebra.is_algebraic.range_eval_eq_root_set_minpoly Algebra.IsAlgebraic.range_eval_eq_rootSet_minpoly
 
```

Comment:
You need to move the `[IsAlgClosed]` on line 401 to the statement of `Algebra.IsAlgebraic.range_eval_eq_rootSet_minpoly` otherwise it is an (unnecessary) hypothesis in the next results. (I don't know how to produce a diff to show that)

Reply:
yeah, I was quite surprised you missed that. I should have known better. At least it's clearer now (:
Sorry for the noise